.eslintrc.json

```diff
@@ -4,7 +4,7 @@
   "rules": {
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
-    "prettier/prettier": "error"
+    "prettier/prettier": "warn"
```

Comment:
Why warn instead of error?

Reply:
I found it distracting when typing to have red squiggly lines everywhere, made me think I have actual errors in my code, I found it useful to differentiate errors and formatting errors visually. But If you want the pipeline to actually fail on prettier errors, I can switch it back :)

Reply:
It would be great if we can make pipeline fail on these errors 😄